charts/incubator/frigate/questions.yaml

```diff
@@ -619,7 +619,7 @@ questions:
                       schema:
                         type: string
                         default: ""
-                        valid_chars: '^[0-9]+(\.[0-9]*)?$'
+                        valid_chars: '^$|^[0-9]+(\.[0-9]*)?$'
```

Comment:
Why are this changing?

Reply:
@stavros-k all these changes were done by you https://github.com/truecharts/charts/pull/7736/commits/79fb40ee15b6bfe514610d66b87822a2b0f11819

Some values are optional as we discovered some months ago. I had reported it, and you made the changes.

Reply:
Not sure what happen with that commit, but I noticed it is implicitly included in the port/incubator #7736 that is currently open.